### PR TITLE
Bugfix/44022 language dropdown cut on arrow click

### PR DESCRIPTION
**Pull request by Farhan-emgage**
language drop down cut on arrow click

### CODE REVIEW

File: src/components/Login/Login.scss [hunk 1]
```diff
@@ -30,5 +30,9 @@
 }
 
 .loginContainer {
-  @include floatRight(8px, 12px)
+ // @include floatRight(8px, 12px);
+  //justify-self: flex-end;
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (Farhan-emgage) [reply src/components/Login/Login.scss:35]: @weegrom as per our discussion we don't need this changes anymore

**Comment by weegrom** [comment]
@Farhan-emgage please make the requested changes on Monday or before that we can get this PR merged.

**Pull request merged**